vocdoni/bridge-ui
# added beta tag

## Short description
This PR adds a beta tag to the header, indicating that this software is in Beta.

## How can it be tested
Open the staging version of the website and check that the header contains the beta tag on every page.

## Tracker ID
https://linear.app/aragon/issue/VOT-85/beta-tag
